jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/util/QuoteUtil.java

```diff
@@ -34,6 +34,7 @@
  * <li>Future HTTPbis ABNF changes for parsing will impact QuotedStringTokenizer</li>
  * </ul>
  * It was decided to keep this implementation separate for the above reasons.
+ * @Deprecated use QuotedCSV
```

Comment:
QuotedCSV cannot handle the quirks in the RFC6455 spec.

Reply:
Really?   My preference would be to update or extend QuotedCSV rather than have an entirely different duplicated parser... unless the requirements are significantly different.

Reply:
@joakime  QuotedCSV is a much newer class that the Quoted tokenizers that existed when you probably wrote QuoteUtil.   I've had a look at RFC6455 and I think it is substantially in line with RFC7230, which is what QuotedCSV is written against.   Specifically both are written against the #() and *() BNF rules and thus have similar treatment of LWS etc.

I'm confident that QuotedCSV is substantially the same and if any quirks do emerge (which I can't initially see), then it can be modified to cope.